### DISCUSSION TITLE
Animated Text Kit selected for GSSOC 2021🎉

### DISCUSSION THREAD

**Discussion by aagarwal1012** [aagarwal1012/Animated-Text-Kit#196] [General]
Hello Everyone👋, 

Animated Text Kit has been selected for GirlScript Summer of Code (GSSOC). It is a three-month-long Open Source program that aims to help beginners get started with Open Source Development during summers. Find more info about GSSOC, https://gssoc.girlscript.tech.

In this program, we mainly focus on expanding the package's domain by adding more animated texts, follow #187 for the previous discussion, and fixing existing issues and bugs. Other ideas are highly welcomed.

cc: @hemilpanchiwala @awhitford @AadumKhor @SirusCodes

**Comment by SirusCodes** [comment]
That's really great! Looking forward for contributions from the participants!
